samples/Samples.Jwt/Controllers/OAuthController.cs

```diff
@@ -9,10 +9,10 @@ public class OAuthController : Controller
     [HttpGet]
     [HttpPost]
     [Route("token")]
-    public IActionResult Token(string grant_type, string client_id, string client_secret)
+    public IActionResult Token([Bind(Prefix = "grant_type")] string grantType, [Bind(Prefix = "client_id")] string clientId, [Bind(Prefix = "client_secret")] string clientSecret)
```

Comment:
This should be reverted. It just makes the code harder to read.

Reply:
Why did you use `snake_case` naming notation here?

Reply:
Part of OAuth spec

Reply:
This should be reverted.  Naming rules should be excluded in `Tests.props`.

Reply:
Reverted. `Tests.props` file is not used for samples. I added `Samples.props`.